bin/reth/Cargo.toml

```diff
@@ -97,14 +97,28 @@ jemallocator = { version = "0.5.0", optional = true }
 jemalloc-ctl = { version = "0.5.0", optional = true }
 
 [features]
-default = ["jemalloc"]
+default = ["jemalloc", "ethereum"]
 jemalloc = ["dep:jemallocator", "dep:jemalloc-ctl"]
 jemalloc-prof = ["jemalloc", "jemallocator?/profiling"]
 min-error-logs = ["tracing/release_max_level_error"]
 min-warn-logs = ["tracing/release_max_level_warn"]
 min-info-logs = ["tracing/release_max_level_info"]
 min-debug-logs = ["tracing/release_max_level_debug"]
 min-trace-logs = ["tracing/release_max_level_trace"]
+optimism = [
+  "reth-primitives/optimism",
+  "reth-revm/optimism",
+  "reth-interfaces/optimism",
+  "reth-rpc/optimism",
+  "reth-rpc-engine-api/optimism",
+  "reth-transaction-pool/optimism",
+  "reth-provider/optimism",
+  "reth-beacon-consensus/optimism",
+  "reth-basic-payload-builder/optimism",
+  "reth-network/optimism",
+  "reth-network-api/optimism"
+]
+ethereum = []
```

Comment:
new feature flags should be documented here
https://github.com/paradigmxyz/reth/blob/d28ef229d9198701bc3700f51ac424530f136577/bin/reth/src/lib.rs#L8-L10

Reply:
Added in https://github.com/anton-rs/op-reth/pull/100